app/helpers/application_helper.rb

```diff
@@ -56,7 +59,7 @@ def us_states
   end
 
   # Append a parameter to a URL string
-  def url_with_param(param, val, url)
+  def append_param_to_url(param, val, url)
```

Comment:
Is this method used anywhere? There are WAY better ways to do this so, if it's not being used, the method should just be removed.

Reply:
In fact, the method doesn't seem to be used anywhere. Let's just remove it.